test/coverage.js

```diff
@@ -232,4 +233,21 @@ describe('Coverage', () => {
             done();
         });
     });
+
+    describe('Clear require cache', () => {
+
+        it('does not reset file coverage', function (done) {
```

Comment:
I ran this without the fix and it passes.  Can you create a test that fails without this fix